.github/workflows/package-tests.yml

```diff
@@ -64,8 +71,15 @@ jobs:
 
       - name: Citus package tests
         run: |
-          export PROJECT_VERSION="${{ github.event.inputs.name }}"
-          [ -z ${PROJECT_VERSION} ]&& export PROJECT_VERSION=10.2.1
+          export PROJECT_VERSION_FROM_PACKAGING="${{ needs.metadata.outputs.citus_version }}"
+          if [[ ! -z ${PROJECT_VERSION_FROM_PACKAGING} ]]
+          then
+            export PROJECT_VERSION=${PROJECT_VERSION_FROM_PACKAGING}
+          else
+            export PROJECT_VERSION="${{ github.event.inputs.prj_ver }}"
+          fi
```

Comment:
can you add some comments to explain in which cases we can (and should) read this from $PROJECT_VERSION_FROM_PACKAGING, and when from ${{ github.event.inputs.prj_ver }} ?

Reply:
Removed it with removing the usage of plugin